### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by ribafs** [andreapollastri/cipi#247] [General]
Thank you very much for this wonderful tool. 
I will create a fork and try to contribute as much as I can.
Cheers

**Comment by andreapollastri** [comment]
Thank You! Any kind of help or support is welcome :)

**Comment by MyBlood** [comment]
And I want to say thank you to everyone involved, everyone has contributed. Even being here is a contribution.

**Comment by arwinvdv** [comment]
@andreapollastri Is there a way we can contribute to v4?

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
